src/pal/src/exception/seh-unwind.cpp

```diff
@@ -195,6 +198,9 @@ BOOL PAL_VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextP
 
     return TRUE;
 }
+#else
+#error don't know how to unwind on this platform
```

Comment:
Sure, I can add that easily enough.  Let me see if  94c197e fixed linux CI, and if so I'll push up the change.